sycl/test-e2e/Graph/Update/whole_update_delay.cpp

```diff
@@ -1,6 +1,7 @@
 // Tests executable graph update by introducing a delay in to the update
 // transactions dependencies to check correctness of behaviour.
-
+// TODO This test is disabled because host-tasks are not supported for graph
+// updates yet.
```

Comment:
Since we specify the update behaviour as blocking I think we can probably just remove this test, since I don't think it's testing anything.

Reply:
Yeah, that makes sense. I deleted the test